.github/workflows/homebrew.yml

```diff
@@ -0,0 +1,39 @@
+name: "Smoke Test Homebrew install"
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  pull_request: # TODO: remove pull_request trigger
```

Comment:
Waiting on IT to create the webhook, after which I'll clean up the TODOs in this file.